docs/reference/inference/put-inference.asciidoc

```diff
@@ -141,11 +173,31 @@ Settings to configure the {infer} task. These settings are specific to the
 .`task_settings` for `text_embedding`
 [%collapsible%closed]
 =====
+`input_type`:::
+(optional, string)
+For `cohere` service only. Specifies the type of input passed to the model.
+Valid values are:
```

Comment:
I also added

- `"classification"`: Used for embeddings passed through a text classifier.
- `"clustering"`: Used for the embeddings run through a clustering algorithm.